### PR TITLE
A recipe can be edited

### PR DESCRIPTION
**Pull request by beccaburns**
A recipe can be edited
=================

[PT-Link](https://www.pivotaltracker.com/story/show/169714851)

```
As a User
In order to improve my own recipe
I would like to be able to change it after creating it
```

## PR Contents
In this PR, a recipe can be edited.

## In this PR
- Create a single component for create and edit recipe called RecipeCU
- Creates integration spec file to test happy and sad path for recipe edit
- Create message to state a successfully edited recipe

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

File: cypress/integration/userCanEditRecipe.spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  it('Successfully', () => {
+    cy.route({
+      method: 'PUT',
+      url: 'http://localhost:3000/v1/recipes/1',
+      status: 201,
+      response: '{ "message": "Your recipe has been updated." }'
+    })
+
+    cy.get('#recipe-1').click({ force: true })
+
```
Comment (clalil) [inline cypress/integration/userCanEditRecipe.spec.js]: Please remove spaces.

File: cypress/integration/userCanEditRecipe.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    cy.get('[name="edit-recipe"]').click()
+
```
Comment (clalil) [inline cypress/integration/userCanEditRecipe.spec.js]: Please remove spaces.

Comment (oliverochman) [reply cypress/integration/userCanEditRecipe.spec.js]: I think this makes it more readable, please have a discussion with the team about this. 
If you all feel the same way, then go ahead and remove them.

**Review by miyeaier — APPROVED** [review]

File: src/App.jsx [hunk 3]
```diff
@@ -14,12 +14,13 @@ class App extends Component {
           <Switch>
             <Route exact path='/' component={ListRecipes} />
             <Route exact path='/recipe/:id' component={SingleRecipe} />
-            <Route exact path="/create" component={CreateRecipe} />
+            <Route exact path="/create" component={RecipeCU} />
           </Switch>
+          <RecipeCU />
```
Comment (clalil) [inline src/App.jsx]: Remove the additional component for RecipeCU, only router is needed

**Review by clalil — APPROVED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Review by yasmineezequiel — APPROVED** [review]

**Pull request merged**